reactor-netty-core/src/main/java/reactor/netty/Metrics.java

```diff
@@ -30,7 +31,7 @@
 public class Metrics {
 	public static final MeterRegistry REGISTRY = io.micrometer.core.instrument.Metrics.globalRegistry;
 	static {
-		REGISTRY.withTimerObservationHandler();
+		REGISTRY.observationConfig().observationHandler(new ReactorNettyTimerObservationHandler(REGISTRY));
```

Comment:
Discussed the change with @marcingrzejszczak, we should the change below


```suggestion
		REGISTRY.observationConfig()
				.observationHandler(new ObservationHandler.FirstMatchingCompositeObservationHandler(
						new ReactorNettyTimerObservationHandler(REGISTRY), new TimerObservationHandler(REGISTRY)));
```